sdk/python/tests/unit/test_datetime.py

```diff
@@ -1,6 +1,19 @@
 # -*- coding: utf-8 -*-
 
+from datetime import datetime
+
+from feast.utils import _utc_now
 
 """
 Test the retirement of datetime.utcnow() function.
 """
+
```

Comment:
Do we need this file? I don't this we should be testing standard lib functions.

Reply:
got it.

Reply:
fair enough.  will delete it.